module/zfs/zio.c

```diff
@@ -97,6 +98,11 @@ int zio_buf_debug_limit = 16384;
 int zio_buf_debug_limit = 0;
 #endif
 
+/*
+ * Dedicated task queue for ZIO artificial delay zinject
+ */
+static taskq_t *zio_delay_taskq;
```

Comment:
Let's use the `system_taskq` for this rather than creating a new dedicated taskq.  This way we can avoid having a bunch of additional idle threads on the systems for this taskq's thread pool.